azure-governance-cloudwise/nested/AutomationRunbooksOMSDashboard/OMSAutomationRunbooksDashboard.json

```diff
@@ -0,0 +1,343 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "omsId": {
+      "type": "string",
+      "metadata": {
+        "description": "Provide the OMS Workspace ID."
+      }
+    },
+    "omsWorkspaceName": {
+      "type": "string",
+      "metadata": {
+        "description": "Provide the OMS Workspace name."
+      }
+    },
+    "omsworkspaceRegion": {
+      "type": "string",
+      "metadata": {
+        "description": "Provide the OMS Workspace Region."
+      }
+    },
+    "omsKey": {
+      "type": "string",
+      "metadata": {
+        "description": "Provide the OMS Workspace KEY."
+      }
+    },
+    "automationAccountName": {
+      "type": "string",
+      "metadata": {
+        "description": "Provide the name of an existing Automation Account with SPN."
+      }
+    },
+    "automationRegion": {
+      "type": "string",
+      "defaultValue": "East US 2",
+      "allowedValues": [
+        "Japan East",
+        "East US 2",
+        "Southeast Asia",
+        "South Central US",
+        "West Europe"
+      ],
+      "metadata": {
+        "description": "Location of the existing Automation Account."
+      }
+    },
+    "_artifactsLocation": {
+      "type": "string",
+      "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/101-webappazure-oms-monitoring",
+      "metadata": {
+        "description": "Path of the template folder."
+      }
+    },
+    "_artifactsLocationSasToken": {
+      "type": "securestring"
+    }
+  },
+  "variables": {
+    "laApiVersion": "2015-11-01-preview",
```

Comment:
This is not really an ARM deploy json file. I've changed the fileextension to omsview and updated the install guide